docs/fernet.rst

```diff
@@ -109,6 +115,34 @@ has support for implementing key rotation via :class:`MultiFernet`.
     the front of the list to start encrypting new messages, and remove old keys
     as they are no longer needed.
 
+    .. method:: rotate(msg, ttl=None)
+
+        Re-encrypts the provided fernet token. If the token has successfully
+        been re-encrypted, then the re-encrypted token will be returned. This
+        will raise an exception if re-encryption fails.
+
+        :param bytes msg: The token to re-encrypt.
+        :param int ttl: Optionally, the number of seconds old a message may be
+                        for it to be valid. If the message is older than
```

Comment:
This is a petty style complaint but I prefer this with a single indent level for multiple lines and not aligned.